tests/test_pypi.py

```diff
@@ -39,3 +40,7 @@ def test_upload_with_custom_path(self, mock_run):
         self.assertEqual(args[2], mock.call(
             'twine upload -u username -p password  custom-dist/*'),)
         self.assertEqual(args[3], mock.call('rm -rf custom-dist'))
+
+    def test_raises_error(self):
```

Comment:
This test name should reflect what it checks, something like `test_raises_error_when_missing_credentials`.